sm64.ld

```diff
@@ -147,8 +147,6 @@ SECTIONS
 
 /* hardcoded symbols to satisfy preliminary link for map parser */
 #ifndef DEBUG_MAP_STACKTRACE
-      parse_map               = MAP_PARSER_ADDRESS;
-      find_function_in_stack  = MAP_PARSER_ADDRESS;
```

Comment:
i know you've mentioned it before, whats the issue here exactly?

Reply:
These functions were dropped from the sm64 preliminary map file because they were specified directly. Their position is not fixed in the end of the data so they changed the address of every function after them. Thankfully, these functions are still being built and just use stub map symbols for the preliminary map (gMapEntries, gMapEntriesSize, _mapDataSegmentRomStart) so it is unnecessary to drop them.